bin/build-with-ssr.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+function fix_server_js_syntax() {
+    sed -i "s|@hmcts/media-viewer;|'@hmcts/media-viewer';|g" ./dist/server.js
+}
+
+function fix_server_js_syntax_mac() {
+    sed -i \"\" "s|@hmcts/media-viewer;|'@hmcts/media-viewer';|g" ./dist/server.js
+}
+
+function fix_colour_scss() {
+    sed -i "s|@error \"Unknown colour \`#{\$colour}\`\";|@return \$colour;|g" ./node_modules/@hmcts/media-viewer/assets/govuk-frontend/helpers/_colour.scss
+}
+
+function fix_colour_scss_mac() {
+    sed -i "" "s|@error \"Unknown colour \`#{\$colour}\`\";|@return \$colour;|g" ./node_modules/@hmcts/media-viewer/assets/govuk-frontend/helpers/_colour.scss
+}
+
+function fix_window_print() {
+    find ./node_modules/@hmcts/media-viewer/ -type f -name '*.js' -exec sed -i "s|printWindow.print();|;|g" {} +;
+}
+
+function fix_window_print_mac() {
+    find ./node_modules/@hmcts/media-viewer/ -type f -name '*.js' -exec sed -i "" "s|printWindow.print();|;|g" {} +;
+}
+
+osName="$(uname -s)"
+echo "Building with ssr on [$osName]."
+if [[ "Darwin" == "$osName" ]];then
+    fix_window_print_mac && fix_colour_scss_mac && npm run build:client-and-server-bundles && npm run webpack:server && fix_server_js_syntax_mac
+else 
```

Comment:
mmm If "$osName" is not "Darwin" we are assuming is windows. But it might not be true. Perhaps better adding another branch for when it's neither mac nor windows where we provide some info message ?

Reply:
Well, this check is actually to differentiate between Mac machines and Linux ones on Jenkins slaves, and is working fine for both environments. We have only 1 windows dev machine, for which this change is not intended.